data-donation-sdk/src/commonMain/kotlin/care/data4life/datadonation/donation/fhir/anonymization/ResearchSubjectAnonymizer.kt

```diff
@@ -16,49 +16,48 @@
 
 package care.data4life.datadonation.donation.fhir.anonymization
 
-import care.data4life.datadonation.donation.fhir.anonymization.model.BlurRule
-import care.data4life.datadonation.donation.program.model.BlurFunction
+import care.data4life.datadonation.donation.fhir.anonymization.model.BlurModelContract.ResearchSubjectBlur
 import care.data4life.hl7.fhir.stu3.model.Period
 import care.data4life.hl7.fhir.stu3.model.ResearchSubject
 
 internal class ResearchSubjectAnonymizer(
-    private val dateTimeSmearer: AnonymizationContract.DateTimeSmearer
+    private val dateTimeConcealer: AnonymizationContract.DateTimeConcealer
 ) : AnonymizationContract.ResearchSubjectAnonymizer {
     private fun blurPeriod(
         period: Period,
-        rule: BlurRule
+        rule: ResearchSubjectBlur
```

Comment:
2 ortogonal things: 
1) if we want to stay with DTOs named `Blur` and our corresponding objects called `BlurRule`, please be consistent with the parameter names too (eg. changing `rule` here to `blur`, happens around the code a few times).
2)  also why not using the `ResearchSubjectBlurRule` here instead of the DTO directly ?  if we want `Blur`s to be only DTO, the rest of the logic should always consistently use the `BlurRule`s after they are resolved.

Reply:
Done